api/config.py

```diff
@@ -74,6 +75,13 @@ class Config(object):
     DISABLE_NAMEREQUEST_NRO_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NRO_UPDATES', 0))
     DISABLE_NAMEREQUEST_SOLR_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_SOLR_UPDATES', 0))
 
+    # NATS
+    NATS_SERVERS = os.getenv('NATS_SERVERS')
+    NATS_CLIENT_NAME = os.getenv('NATS_CLIENT_NAME')
+    NATS_CLUSTER_ID = os.getenv('NATS_CLUSTER_ID')
+    NATS_QUEUE = os.getenv('NATS_QUEUE')
+    NATS_NR_STATE_SUBJECT = os.getenv('NATS_NR_STATE_SUBJECT', 'namerequest.state')
```

Comment:
default should be something more like namex.event

Reply:
ok. Done.